components/CtaLink/src/index.scss

```diff
@@ -58,7 +58,7 @@
 
 @supports (-webkit-line-clamp: 2) {
   .denhaag-cta-link__excerpt {
-    display: box;
+    display: flex;
```

Comment:
Somehow lijkt dit toch een side-effect te hebben in de whitespace tussen teksten 😢 dus even kijken hoe die magic whitespace weer terug komt, misschien met padding of een gap ofzo...